test/DynamicHelpTest.cs

```diff
@@ -0,0 +1,146 @@
+            {
+                PSObject[] descDetails = new PSObject[1];
+                descDetails[0] = new PSObject();
+                descDetails[0].Members.Add(new PSNoteProperty("Text", null));
+            }
+            else if (string.Equals(commandName, "Get-Date", StringComparison.OrdinalIgnoreCase) && string.Equals(parameterName, "Date", StringComparison.OrdinalIgnoreCase))
+            {
+                PSObject[] descDetails = new PSObject[1];
+                descDetails[0] = new PSObject();
+                descDetails[0].Members.Add(new PSNoteProperty("Text", descText));
+
+                var np = new PSNoteProperty("name", "System.Datetime");
+                np.Value = "System.Datetime";
+
+                var typeName = new PSObject(np);
+
+                paramHelp.Members.Add(new PSNoteProperty("Description", descDetails));
+                paramHelp.Members.Add(new PSNoteProperty("Name", "Date"));
+                paramHelp.Members.Add(new PSNoteProperty("type", typeName));
+                paramHelp.Members.Add(new PSNoteProperty("required", "false"));
+                paramHelp.Members.Add(new PSNoteProperty("position", "0"));
+                paramHelp.Members.Add(new PSNoteProperty("defaultValue", "None"));
+                paramHelp.Members.Add(new PSNoteProperty("pipelineInput", "True (ByPropertyName, ByValue)"));
+                paramHelp.Members.Add(new PSNoteProperty("globbing", "false"));
+            }
+
+            return paramHelp;
+        }
+
+        [SkippableFact]
+        public void DynHelp_GetFullHelp()
+        {
+            TestSetup(KeyMode.Cmd);
+            Test("Get-Date", Keys(
+                "Get-Date", _.F1,
+                CheckThat(() => Assert.Equal(fullHelp, _mockedMethods.helpContentRendered)),
+                _.Enter
+                ));
+        }
+
+        [SkippableFact]
+        public void DynHelp_GetParameterHelp()
+        {
+            TestSetup(KeyMode.Cmd);
+            string emptyLine = new string(' ', _console.BufferWidth);
+
+            Test("Get-Date -Date", Keys(
+                "Get-Date -Date", _.Alt_h,
+                CheckThat(() => AssertScreenIs(9,
+                TokenClassification.Command, "Get-Date",
+                TokenClassification.None, " ",
+                TokenClassification.Parameter, "-Date", NextLine,
+                emptyLine,
+                TokenClassification.None, $"-Date <name>", NextLine,
+                emptyLine,
+                TokenClassification.None, "DESC: Specifies a date and time. Time is optional and if not", NextLine, " specified, returns 00:00:00.", NextLine,
+                TokenClassification.None, "Required: false, Position: 0, Default Value: None, Pipeline ", NextLine, "Input: True (ByPropertyName, ByValue), WildCard: false")),
+                _.Enter,
+                _.Enter
```

Comment:
No test to verify that `MultilineDisplayBlock.Clear()` works as expected. Please add test to cover that, basically, do a `AssertScreenIs` again to valdiate if screen was cleared.

Reply:
Added Clear verification to this test.